app/services/migration/migrators/application_not_in_ecf.rb

```diff
@@ -0,0 +1,15 @@
+module Migration::Migrators
+  class ApplicationNotInEcf < Base
+    def call
+      migrate(applications, :application_not_in_ecf) do |application|
```

Comment:
I was wondering if we could do those in the same migrator if possible, or do they have to be separate?

Reply:
Hi @cwrw I've tried to do all in the same migrator but couldn't find an easy way of doing it as we use `in_batches` in the migrator which accepts a `ActiveRecord::Relation` query but technically `Application` and `Migration::Ecf::NpqApplication` are different models, so decided to split in 2 migrators. Easier to test as well and see results.